index.js

```diff
@@ -24,8 +24,6 @@ module.exports = {
       mergedTrees = mergeTrees([this.app.trees.app]);
```

Comment:
Agreed, seems not needed. This PR was just the minimal changes to get this working (and remove wasted double babel processing etc).